scripts/minikube-start.sh

```diff
@@ -6,9 +6,8 @@ require_tools minikube
 : "${VM_CPUS:=4}"
 : "${VM_MEMORY:=16384}"
 : "${VM_DISK_SIZE:=120g}"
-
-: "${MINIKUBE_ISO_VERSION:=0.1.8}"
-: "${MINIKUBE_ISO_URL:=https://github.com/f0rmiga/opensuse-minikube-image/releases/download/v${MINIKUBE_ISO_VERSION}/minikube-openSUSE.x86_64-${MINIKUBE_ISO_VERSION}.iso}"
+: "${VM_DRIVER:=virtualbox}"
```

Comment:
Why does Minikube need a vm environment and what happens if Virtualbox is not installed?

Reply:
These are the default configuration on minikube which works. Users should install something like Virtualbox or docker or hyperkit whichever they prefer.

Reply:
Minikube recommends Docker on there site as preferred option (https://minikube.sigs.k8s.io/docs/drivers/), but if this is a special use case, I do not mind.

Reply:
I use `minikube start --docker-opt storage-driver=overlay2` to run minikube on Docker. The `docker-opt` addresses a performance warning you get during startup.